electrum/lnutil.py

```diff
@@ -414,10 +413,7 @@ def formatted_tuple(self):
 
 
 class LightningError(Exception): pass
-class LightningPeerConnectionClosed(LightningError): pass
 class UnableToDeriveSecret(LightningError): pass
-class HandshakeFailed(LightningError): pass
-class ConnStringFormatError(LightningError): pass
 class RemoteMisbehaving(LightningError): pass
```

Comment:
note: Hmm looks like we never rely on these being subclasses of `LightningError`